src/easy-peasy.js

```diff
@@ -4,15 +4,25 @@ import {
   createStore as reduxCreateStore,
 } from 'redux'
 import memoizeOne from 'memoize-one'
-import produce from 'immer'
+import produce, { setAutoFreeze } from 'immer'
 import thunk from 'redux-thunk'
 import { isStateObject } from './lib'
 
-const effectSymbol = '__effect__'
-const selectSymbol = '__select__'
-const selectDependenciesSymbol = '__selectDependencies__'
-const selectStateSymbol = '__selectState__'
-const reducerSymbol = '__reducer__'
+setAutoFreeze(false)
```

Comment:
Interesting. I had no idea about this. Perhaps we should tie the value to the dev boolean config of the store?

Reply:
I agree, that would be better